### PR TITLE
Objects Automated Test - LPS-141681 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-141681

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 1]
```diff
@@ -4521,12 +4521,38 @@ definition {
...
+			relationshipType = "oneToMany");
+
+		ObjectAdmin.openObjectAdmin();
 
-		// TODO LPS-141681 ObjectNameDisplayedOnTheRelationshipTab pending implementation
+		ObjectPortlet.selectCustomObject(label = "Custom Object A");
+
+		ObjectAdmin.gotoRelationshipsTab();
 
+		CreateObject.assertRelatedObject(relatedObject = "CustomObject");
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:4555]: Hi @andreefarias 

You need specific the object correct, I believe that you need to use `CustomObjectB`

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @andreefarias ,

Please, check my comments and make the necessary changes.

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase (hunk 1 above)
Comment (andreefarias) [reply modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase]: Fixed!

**Review by evanilson-santana — APPROVED** [review]

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**